pkg/storage/unified/resource/access.go

```diff
@@ -199,7 +198,7 @@ func (c authzLimitedClient) Compile(ctx context.Context, id claims.AuthInfo, req
 	}
 	checker, err := c.client.Compile(ctx, id, req)
 	if err != nil {
-		c.logger.Error("Compile", "group", req.Group, "resource", req.Resource, "error", err, "traceid", tracing.TraceIDFromContext(ctx, false))
+		c.logger.Error("Compile", "group", req.Group, "resource", req.Resource, "error", err, "traceid", trace.SpanContextFromContext(ctx))
```

Comment:
same suggestion as above
```suggestion
		c.logger.Error("Compile", "group", req.Group, "resource", req.Resource, "error", err, "traceid", trace.SpanContextFromContext(ctx).TraceID.String())
```

Reply:
oh, good catch! Didn't notice `SpanContextFromContext` returns a stuct! :)